specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-12-01/bms.json

```diff
@@ -5454,6 +5454,21 @@
           "description": "The oldest backup copy available for this backup item.",
           "type": "string"
         },
+        "oldestRecoveryPointInVault": {
```

Comment:
>oldestRecoveryPointInVault [](http://example.com/codeflow?start=9&length=26)

The API versioning comment applies to this as well

Reply:
Replied above